crypto/serializer/deserializer_lib.c

```diff
@@ -181,7 +181,7 @@ int OSSL_DESERIALIZER_CTX_add_extra(OSSL_DESERIALIZER_CTX *ctx,
     size_t w_prev_start, w_prev_end; /* "previous" deserializers */
     size_t w_new_start, w_new_end;   /* "new" deserializers */
     size_t count; /* Used to calculate how many were added in each iteration */
```

Comment:
Looks like you should also set count to zero here..

You have a continue inside the do loop (which will then go to the while check with count uninited

Reply:
Uhmmm, the continues are inside the for loop that is inside the do loop, no?

Reply:
yep you are right..